hedera-node/test-clients/build.gradle.kts

```diff
@@ -71,12 +71,12 @@ eetModuleInfo {
 tasks.itest {
     systemProperty("junit.jupiter.execution.parallel.enabled", false)
     systemProperty("TAG", "services-node:" + project.version)
-    systemProperty("networkWorkspaceDir", File(project.buildDir, "network/itest"))
+    systemProperty("networkWorkspaceDir", layout.buildDirectory.dir("network/itest").get().asFile)
```

Comment:
`project.buildDir` is now deprecated and `layout.buildDirectory` should be used consistently instead.